packages/core/src/connectivity/scp-cf/destination/destination.ts

```diff
@@ -55,7 +55,40 @@ export function parseDestination(
     }
   );
 
-  return sanitizeDestination(destination);
+  const additionalHeadersAndQueryParameters =
+    getAdditionalHeadersAndQueryParameters(destinationConfig);
+
+  return sanitizeDestination({
+    ...destination,
+    ...additionalHeadersAndQueryParameters
+  });
+}
+
+function getAdditionalHeadersAndQueryParameters(
```

Comment:
I do not get where the query parameters are handled.

Reply:
I only handle them for the OData case, totally forgot about the http-client. I will try to get this in here, might be more complex than expected, because we do funny stuff with query parameters for OData...